mkdocs.yml

```diff
@@ -30,6 +30,7 @@ nav:
         - './getting-started/Ch-GettingStartedGoDevelopers.md'
         - './getting-started/Ch-GettingStartedCDevelopers.md'
         - './getting-started/Ch-GettingStartedHybrid.md'
+        - './getting-started/Ch-GettingStartedDTOValidation.md'
```

Comment:
Should this be in the Users section as they will be the ones confused by the character restrictions when adding new objects.

Reply:
Moved to user section.